### PR TITLE
Added Contribution Guide, modified README

### PR DESCRIPTION
**Pull request by DiogoRDuarte**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
```
Comment (ritacosta0) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]: I suggest a slightly more formal phrasing. Something like: "Describe your proposed solution"

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+A clear and concise description of what you want to happen.
```
Comment (ritacosta0) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]: I think the idea here is more "A clear and concise description of the feature you are proposing and what it should do"

**Review by ritacosta0 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (DiogoRDuarte) [reply .github/ISSUE_TEMPLATE/feature_request.md]: done ✅

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (DiogoRDuarte) [reply .github/ISSUE_TEMPLATE/feature_request.md]: done ✅

**Review by ritacosta0 — APPROVED** [review]

**Pull request merged**